apps/api/src/database/migrations/20240112144180_remove_city_town_junction_from_zoomlevels_table/migration.sql

```diff
@@ -0,0 +1,24 @@
+/*
+  Delete the zoom levels for town, city and junction
+  Update those application details that referenced the above to a zoom level of district
+
+*/
+BEGIN TRY
+
+BEGIN TRAN;
+
+UPDATE [dbo].[ApplicationDetails] SET zoomLevelId = (SELECT [id] FROM [dbo].[ZoomLevel] WHERE name = 'district') WHERE zoomLevelId in (SELECT [id] FROM [dbo].[ZoomLevel] WHERE name = 'town' or name = 'city' or name = 'junction');
+DELETE FROM [dbo].[ZoomLevel] WHERE name = 'town' or name = 'city' or name = 'junction'
```

Comment:
I may have used the IN keyword, eg DELETE FROM [dbo].[ZoomLevel] WHERE name IN ('town', 'city', 'junction')

and similar for the update, but this is just a comment, the code is fine!

Reply:
Yes, good idea, I'll remember for next time.